itests/ccupgrade_test.go

```diff
@@ -3,21 +3,20 @@ package itests
 import (
 	"context"
 	"fmt"
-	"sync/atomic"
 	"testing"
 	"time"
 
 	"github.com/filecoin-project/lotus/itests/kit"
```

Comment:
Why do we still have this import here? 🤔

Reply:
Fixed